src/cider/tests/utils/CiderArrowChecker.cpp

```diff
@@ -321,6 +321,9 @@ bool checkOneScalarArrowEqual(const struct ArrowArray* expect_array,
       return checkArrowBufferFp<float>(expect_array, actual_array);
     case 'g':
       return checkArrowBufferFp<double>(expect_array, actual_array);
+    case 'd':
+      // workaround for non-groupby agg test
```

Comment:
What's the problem for current agg result comparison? output types are different from duckdb and cider?

Reply:
yes, duckdb return decimal  

> possible to add some test start from CiderNextgenTestBase?

has one in NextgenCompilerTest

Reply:
okay. This is weird...